examples/pysdl2.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/python2
+
+#
+# Simple SDL2 / cefpython3 example.
+#
+# Only handles mouse events but could be extended to handle others.
+#
+# Requires pysdl2 (and SDL2 library).
+#
+# Tested under Fedora Linux (x86_64).
+#
+# by Neil Munday (www.mundayweb.com)
```

Comment:
Do you agree to release your code under BSD-3 new license that cefpython uses? You can add your name to the Authors file in the root directory along with your email or website. The copyright messages in cefpython source files are using this template:

```
# Copyright (c) {YEAR_OF_CREATION} CEF Python, see the Authors file.
# All rights reserved. Licensed under BSD 3-clause license.
# Project website: https://github.com/cztomczak/cefpython
```

The examples from the examples/ directory do not contain any copyright messages and this is intentional, because these examples are intended to be edited freely by users.

When your PR is accepted you will also appear on the Contributors page:
https://github.com/cztomczak/cefpython/graphs/contributors